### PR TITLE
Removed SSH server and CMD, tailing dummy file to keep container alive.

### PR DESCRIPTION
**Pull request by indyaah**
Moved app directory under jhipster home.

Removed need for apt-add-repository and maven from ubuntu source.

fixes jhipster/jhipster-docker#11 if accepted.
Probably fixes jhipster/generator-jhipster#2203 as well.

### CODE REVIEW

**Comment by indyaah** [comment]
Tried and test ok with following command.
docker run --name jhipster -w /home/jhipster/jhipster-sample-app -v ~/jhipster:/jhipster -v ~/.m2:/home/jhipster/.m2:rw -u "jhipster" -p 8080:8080 -p 3000:3000 -p 3001:3001 -t jdubois/jhipster-docker mvn

File: Dockerfile [hunk 1]
```diff
@@ -50,10 +45,13 @@ RUN cd /home/jhipster && \
     rm v2.23.0.zip
 RUN cd /home/jhipster/jhipster-sample-app-2.23.0 && npm install
 RUN cd /home && chown -R jhipster:jhipster /home/jhipster
-RUN cd /home/jhipster/jhipster-sample-app-2.23.0 && sudo -u jhipster mvn dependency:go-offline
+RUN cd /home/jhipster/jhipster-sample-app-2.23.0
+# RUN sudo -u jhipster mvn dependency:go-offline
```
Comment (pascalgrimaud) [inline Dockerfile]: maybe remove both ?

**Comment by indyaah** [comment]
Also the difference in generated image size is significant.

**Original image**
REPOSITORY                                          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
jdubois/jhipster-docker                             latest              e2938f4b2001        2 weeks ago         1.84 GB

**New image**
REPOSITORY                                          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
indyaah/jhipster-docker                             latest              1a2960507122        3 hours ago         1.439 GB

File: Dockerfile (hunk 1 above)
Comment (indyaah) [inline Dockerfile]: Yeah, seems like a silly miss. I am correcting it right now.

**Comment by pascalgrimaud** [comment]
LGTM and also fix https://github.com/jhipster/generator-jhipster/issues/2203

File: Dockerfile [hunk 2]
```diff
@@ -1,36 +1,32 @@
 # DOCKER-VERSION 0.7.1
-FROM      ubuntu:14.04
+FROM      ubuntu:trusty
 MAINTAINER Julien Dubois <julien.dubois@gmail.com>
 
-# make sure the package repository is up to date
-RUN echo "deb http://archive.ubuntu.com/ubuntu trusty main universe" > /etc/apt/sources.list
-RUN apt-get -y update
+ENV JAVA_VER 8
```
Comment (jdubois) [inline Dockerfile]: Why not "JAVA_VERSION", to be consistent with "MAVEN_VERSION"?

Comment (indyaah) [inline Dockerfile]: @jdubois : Updated JAVA_VER to JAVA_VERSION to be consistent.

**Pull request merged**